pkg/common/utils/utils.go

```diff
@@ -539,179 +330,30 @@ func PatchVirtualMachine(ctx context.Context, vmOperatorClient client.Client,
 	return nil
 }
 
-func UpdateVirtualMachine(ctx context.Context, vmOperatorClient client.Client,
-	vm *vmoperatorv1alpha5.VirtualMachine) error {
-	vmV1alpha1 := &vmoperatorv1alpha1.VirtualMachine{}
-	vmV1alpha2 := &vmoperatorv1alpha2.VirtualMachine{}
-	vmV1alpha3 := &vmoperatorv1alpha3.VirtualMachine{}
-	vmV1alpha4 := &vmoperatorv1alpha4.VirtualMachine{}
+// GetVirtualMachineList get lists of all the virtual machines
+func GetVirtualMachineList(ctx context.Context, namespace string,
```

Comment:
What's the difference between ListVirtualMachines and GetVirtualMachineList? why do we have duplicates?

Reply:
removed GetVirtualMachineList as now we are not fetching crd version